$MirOS: ports/net/djbdns/patches/patch-man_axfr-get_8,v 1.1 2005/07/02 10:44:14 tg Exp $
--- man/axfr-get.8.orig	Thu Oct 23 07:43:53 2003
+++ man/axfr-get.8	Sun Jan 15 12:56:07 2006
@@ -3,11 +3,6 @@
 .SH NAME
 axfr-get \- a DNS zone-transfer client.
 
-It sends a zone-transfer request
-in DNS-over-TCP format to descriptor 7,
-reads the results from descriptor 6,
-and saves the results in a file.
-
 .SH SYNOPSIS
 Normally 
 .B axfr-get
@@ -21,6 +16,11 @@ which sets up descriptors 6 and 7 as a T
 .I fn.tmp
 
 .SH DESCRIPTION
+It sends a zone-transfer request
+in DNS-over-TCP format to descriptor 7,
+reads the results from descriptor 6,
+and saves the results in a file.
+
 .B axfr-get
 performs a zone transfer for domain 
 .IR z .
